lib/tasks/evm_application.rb

```diff
@@ -102,11 +102,12 @@ def self.output_workers_status(servers)
            w.miq_server_id,
            w.queue_name || w.uri,
            w.started_on && w.started_on.iso8601,
-           w.last_heartbeat && w.last_heartbeat.iso8601]
+           w.last_heartbeat && w.last_heartbeat.iso8601,
+           ActionView::Base.new.number_to_human_size(w.proportional_set_size || w.memory_usage) || ""]
```

Comment:
@Fryguy Ah, so we are back to the really gross ternary I suggested here:  https://github.com/ManageIQ/manageiq/pull/15375#discussion_r122061822

:trollface: